src/utils/data-formatter.js

```diff
@@ -35,6 +35,8 @@ export function formatDateForInput(dateString) {
 }
 
 export function prepareResourcesContent(resourcesContent) {
+  if (!resourcesContent)  return
+  
   return resourcesContent.map((resource) => ({
     id: resource.resource ? resource.resource.id : resource.id,
     quantity: resource.quantity,
```

Comment:
In this case assuming we want to deal with arrays I think a better approach is to use default value like this:
```
export function prepareResourcesContent(resourcesContent = []) {
  return resourcesContent.map((resource) => ({
    id: resource.resource ? resource.resource.id : resource.id,
    quantity: resource.quantity,
  }));
}
```